src/rexsdesktop/model/Db.java

```diff
@@ -1700,10 +1700,10 @@ public boolean eliminarFotoPerfil(int idUsuario) {
      */
     public void NumUsuarios() {
         try {
-            int tipoU = CurrentUser.idTipoUsuario;
+            int tipoU = CurrentUser.getIdTipoUsuario();
             String sql;
 
-            sql = "select COUNT(idUsuario) from usuario where idTipoUsuario != " + 1 + "and idUsuario != " + CurrentUser.idUsuario;
+            sql = "select COUNT(idUsuario) from usuario where idTipoUsuario != " + 1 + "and idUsuario != " + CurrentUser.getIdUsuario();

```

Comment:
I know this is out of the scope and hopefully we could see this later, but I just want let you know this isn't the best way to communicate with the database.

Reply:
Yes I know, I didn't write that query, I would normally use a prepared statement instead of concatenation.